src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -624,6 +625,13 @@ private void preserveMethodDecl(ResolvedMethodDeclaration decl) {
     catch (UnsolvedSymbolException e) {
       return;
     }
+
+    for (int i = 0; i < decl.getNumberOfParams(); ++i) {
```

Comment:
We've already have the `visit(Parameter, Void)` to handle parameters of target members. I think we can skip this part.

Reply:
Without this change, the test `OtherPackageArrayTest` fails: the class `Type` isn't created, and the output isn't compilable.

I suspect that there might be another bug in our implementation of `visit(Parameter, Void)`. I don't see a real problem with having redundancy here, but we should also try to figure out why that method doesn't work as expected in this case.